src/components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useContext } from 'react'
+import './Navigation.css'
+import activeModalContext from '../../utils/activeModal'
+
+function Navigation() {
+
+  const {activeModal, setActiveModal} = React.useContext(activeModalContext);
+
+  const handleAbout = () => {
+    setActiveModal("about");
+  }
+  return (
+    <>
```

Comment:
`<>` empty tags are not needed if you have a common tag that wraps the whole markup of the component